### PR TITLE
Add database seeding script (including S3 seeding)

### PR DESCRIPTION
**Pull request by sjamcox**

### CODE REVIEW

File: server/database/schema.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+mongoose.connect(process.env.MONGO_URI, {
+  useNewUrlParser: true,
+  useUnifiedTopology: true
+})
+  .catch(err => console.error(err));
+
+const userSchema = new mongoose.Schema({
+  userId: Number,
+  name: String,
+  joinDate: String,
```
Comment (rgagnon2) [inline server/database/schema.js:11]: Something I learned is that mongoose has a 'Date' type, which is aligned with the javascript date objects: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date
It makes handling dates and manipulating them a lot easier! Even if you store the date as a string, you can do a new Date(strDate) in your code later to convert it back.

File: server/database/seed.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
+require('dotenv').config();
+const { generatePhoto } = require('./helpers');
+const faker = require('faker');
+const AWS = require('./aws.js');
+const { mongoose, User } = require('./schema.js');
+
+const languages = ['English', 'Spanish', 'French', 'Portuguese', 'German', 'Italian', 'Cambodian', 'Thai', 'Shyriiwook'];
```
Comment (rgagnon2) [inline server/database/seed.js:7]: LOL @ Shyriiwook

**Review by rgagnon2 — APPROVED** [review]
Impressive thoroughness with your testing, and nice use of 'faker'!

File: server/database/schema.js (hunk 1 above)
Comment (sjamcox) [reply server/database/schema.js:11]: Nice! I'll have to take a look. Thanks for the suggestion.

**Pull request merged**